sdk/eventhub/event-hubs/src/models/private.ts

```diff
@@ -40,7 +40,7 @@ export type OperationNames = "getEventHubProperties" | "getPartitionIds" | "getP
  * @internal
  * @ignore
  */
-export interface CommonEventProcessorOptions  // make the 'maxBatchSize', 'maxWaitTimeInSeconds', 'ownerLevel' fields required extends // for our internal classes (these are optional for external users)
+export interface CommonEventProcessorOptions  // make the 'maxBatchSize', 'maxWaitTimeInSeconds', 'ownerLevel' fields required extends for our internal classes (these are optional for external users)
```

Comment:
I think we can just ditch this comment altogether - I'm not even sure why it's there.

Reply:
Sounds good to me!

Reply:
Updated!